### DISCUSSION TITLE
无法获取到Android端智能修复画质选项（~~话说这玩意是不是用npu超分啊~~ web端也有，看来不是

### DISCUSSION THREAD

**Discussion by TPAXcc** [ywmoyue/biliuwp-lite#833] [feature]
### 问题描述

http://b23.tv/ss48267

### 描述解决方案

.

### 代码贡献

否

**Comment by ywmoyue** [comment]
相关文档，后续做支持 https://github.com/SocialSisterYi/bilibili-API-collect/pull/1063

**Comment by ywmoyue** [comment]
https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.7.2.302

**Comment by TPAXcc** [comment]
> https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.7.2.302

好像不太对，是720p的选项

**Comment by TPAXcc** [comment]
> > https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.7.2.302
> 
> 好像不太对，是720p的选项

难道这B玩意还要大会员？

**Comment by TPAXcc** [comment]
还真是...那我这测不了了大会员过期了
